docs/reference/commandline/volume_create.md

```diff
@@ -0,0 +1,41 @@
+<!--[metadata]>
++++
+title = "volume create"
+description = "The volume create command description and usage"
+keywords = ["volume, create"]
+[menu.main]
+parent = "smn_cli"
+weight=1
++++
+<![end-metadata]-->
+
+# volume create
+
+  Usage: docker volume create [OPTIONS]
+
+  Create a volume
+
+    -d, --driver=local    Specify volume driver name
+    --help=false          Print usage
+    --name=               Sepcify volume name
+    -o, --opt=map[]       Set driver specific options
+
+The `docker volume create` command creates a new volume that can be consumed by
+a container to store data in. If a name is not specified, a random ID will be
+generated. A container can be configured to use a created volume:
+
+  $ docker volume create --name hello
+  $ docker run -d -v hello:/world busybox ls /world
+
+Multiple containers can use the same volume.
+
+## Driver specific options
+
+Some volume drivers may take options to customize the volume being created,
+these options may be passed through using the `-o` or `--opt` flags:
+
+  $ docker volume create --opt size=10G
```

Comment:
I think we should add a "fake" driver here, otherwise people will assume the default driver supports this option (even though there's a note below), also, mention that `--opt` can be set multiple times, e.g.;

```bash
$ docker volume create --driver some-driver --opt size=smaller-on-the-outside --opt color=police-blue
```